src/components/AppComponents/TimeoutPopup/unauthTimeOut.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import TimeoutPopup from '.';
+import Button from '../../BaseComponents/Button/Button';
+// import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+export default function UnauthTimeOut({ show, modalId, primaryHandler, secondaryHandler }) {
+  // Welsh translation to be added later, not part of story
+  //   const { t } = useTranslation();
+
+  return (
+    <TimeoutPopup show={show} handleClose={() => {}} staySignedIn={primaryHandler} id={modalId}>
+      <div>
+        <h1 id='govuk-timeout-heading' className='govuk-heading-m push--top'>
+          For your security
```

Comment:
add the text content on this page to translation json, including the button labels